distribution/deb/src/test/java/org/elasticsearch/test/rest/ClientYamlTestSuiteIT.java

```diff
@@ -20,18 +20,21 @@
 package org.elasticsearch.test.rest;
 
 import com.carrotsearch.randomizedtesting.annotations.ParametersFactory;
-import org.elasticsearch.test.rest.parser.RestTestParseException;
+
+import org.elasticsearch.test.rest.yaml.ClientYamlTestCandidate;
+import org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase;
+import org.elasticsearch.test.rest.yaml.parser.ClientYamlTestParseException;
 
 import java.io.IOException;
 
 /** Rest integration test. Runs against a cluster started by {@code gradle integTest} */
-public class RestIT extends ESClientYamlSuiteTestCase {
-    public RestIT(RestTestCandidate testCandidate) {
+public class ClientYamlTestSuiteIT extends ESClientYamlSuiteTestCase {
```

Comment:
We were sort of, half way, standardizing on calling them `RestIT` which is no longer the appropriate name. I like having them standardized, especially if we're going to add more `IT`s that aren't the yaml stuff. I think it doesn't hurt anything to have them all with the same name because they are all in different projects.